scipy/sparse/matrix_io.py

```diff
@@ -0,0 +1,149 @@
+    Store sparse matrix to disk, and load it again:
+
+    >>> import scipy.sparse
+    >>> sparse_matrix = scipy.sparse.csc_matrix(np.array([[0, 0, 3], [4, 0, 0]]))
+    >>> sparse_matrix
+    <2x3 sparse matrix of type '<type 'numpy.int64'>'
+       with 2 stored elements in Compressed Sparse Column format>
+    >>> sparse_matrix.todense()
+    matrix([[0, 0, 3],
+            [4, 0, 0]], dtype=int64)
+
+    >>> scipy.sparse.save('/tmp/sparse_matrix.npz', sparse_matrix)
+    >>> sparse_matrix = scipy.sparse.load('/tmp/sparse_matrix.npz')
+
+    >>> sparse_matrix
+    <2x3 sparse matrix of type '<type 'numpy.int64'>'
+       with 2 stored elements in Compressed Sparse Column format>
+    >>> sparse_matrix.todense()
+    matrix([[0, 0, 3],
+            [4, 0, 0]], dtype=int64)
+    """
+
+    if matrix.format in ('csc', 'csr', 'bsr'):
+        arrays_dict = {'format': np.array(matrix.format), 'shape': np.array(matrix.shape), 'data': matrix.data, 'indices': matrix.indices, 'indptr': matrix.indptr}
+    elif matrix.format == 'dia':
+        arrays_dict = {'format': np.array(matrix.format), 'shape': np.array(matrix.shape), 'data': matrix.data, 'offsets': matrix.offsets}
+    elif matrix.format == 'coo':
+        arrays_dict = {'format': np.array(matrix.format), 'shape': np.array(matrix.shape), 'data': matrix.data, 'row': matrix.row, 'col': matrix.col}
+    else:
+        raise NotImplementedError('Save is not implemented for sparse matrix of format {}.'.format(matrix.format))
+    
+    if compressed:
+        np.savez_compressed(file, **arrays_dict)
+    else:
+        np.savez(file, **arrays_dict)
+
+
+def load(file):
+    """ Load a sparse matrix.
+    
+    Load a sparse matrix from a file in ``.npz`` format.
+
+    Parameters
+    ----------
+    file : str or file
+        Either the file name (string) or an open file (file-like object)
+        where the data will be loaded.
+
+    Returns
+    -------
+    result : _cs_matrix
+        A sparse matrix of this class containing the loaded data.
+
+    Raises
+    ------
+    IOError
+        If the input file does not exist or cannot be read.
+
+    See Also
+    --------
+    numpy.load
+
+    Examples
+    --------
+    Store sparse matrix to disk, and load it again:
+
+    >>> import scipy.sparse
+    >>> sparse_matrix = scipy.sparse.csc_matrix(np.array([[0, 0, 3], [4, 0, 0]]))
+    >>> sparse_matrix
+    <2x3 sparse matrix of type '<type 'numpy.int64'>'
+       with 2 stored elements in Compressed Sparse Column format>
+    >>> sparse_matrix.todense()
+    matrix([[0, 0, 3],
+            [4, 0, 0]], dtype=int64)
+
+    >>> scipy.sparse.save('/tmp/sparse_matrix.npz', sparse_matrix)
+    >>> sparse_matrix = scipy.sparse.load('/tmp/sparse_matrix.npz')
+
+    >>> sparse_matrix
+    <2x3 sparse matrix of type '<type 'numpy.int64'>'
+       with 2 stored elements in Compressed Sparse Column format>
+    >>> sparse_matrix.todense()
+    matrix([[0, 0, 3],
+            [4, 0, 0]], dtype=int64)
```

Comment:
I removed the example in ``load`` and added ``load`` and ``save`` vice versa in the ``see also`` section.